### DISCUSSION TITLE
Remove all motion features

### DISCUSSION THREAD

**Discussion by silvanheller** [vitrivr/cineast#277] [General]
In the spirit of keeping Cineast fresh and relevant, I was wondering whether we should remove some / all of the motion features given that they have not been used for multiple years, I'm not sure anybody has tested frontend capabilities in years and I'm not sure they're particularly useful, especially in comparison with newer features.

**Comment by lucaro** [comment]
While they are admittedly not very useful, I'd argue just removing them for the sake of it is wasted effort, as long as they don't hurt anyone. Whenever there is a next round of change on the UI or the API or any other event that would potentially require maintenance on them, they are certainly a good candidate for removal. Since they pre-date cheap semantic object tracking, it would also be worth considering to re-imagine the way to query for motion at some point, now that much more powerful tracking methods are available.

**Reply by silvanheller** [reply]
I'm actually in the process of cleaning up the codebase from ADAMpro-related code (+ old Import-Code) and that's why the thought crossed my mind. 

I'll make a proposal in the cleanup PR, and we'll see if others have an opinion then or here in this thread.

**Reply by lucaro** [reply]
If you're cleaning up anyway, sure, why not.
